app/functoria_app.ml

```diff
@@ -113,7 +114,7 @@ let pp_dump_info module_name fmt i =
     "%s.{@ name = %S;@ \
      @[<v 2>packages = %a@]@ ;@ @[<v 2>libraries = %a@]@ }"
     module_name (Info.name i)
-    pp_packages (Info.packages i)
+    pp_packages (Info.package_names i)
```

Comment:
`pp_dump_info` is used to expose the content of `Info` to the underlying unikernel, in order to do things like an unikernel that lists the packages and the version it uses. It dumps an ml files with placeholder like `%{functoria:version}%` and then call `opam config subst` to substitute them.

That probably won't really work well anymore, so we need to move `opam config subst` to the "make depend" target.
Since you have a bit more information now, we could enrich the datastructure in functoria-runtime with packages and dump that.